v0/destinations/ga/transform.js

```diff
@@ -934,7 +934,7 @@ function processSingleMessage(message, destination) {
           : ConfigCategory.NON_ECOM;
         category.hitType = "event";
         customParams.ni =
-          message?.properties.nonInteraction !== undefined
+          message?.properties?.nonInteraction !== undefined
             ? message.properties.nonInteraction
             : 1;
```

Comment:
```suggestion
          isDefinedAndNotNull(message?.properties?.nonInteraction)
            ? message.properties.nonInteraction
            : 1;
```

Reply:
Maybe you need to import that utility method.

Reply:
We don't need to check the validity of `message` (i.e root level)